SU-HSDO/suhumsci
# shs-44: firefinch color pairing for traditional theme

## READY FOR REVIEW

## Summary
This adds a new color pairing, Firefinch, to the traditional theme.

## Steps to Test
1. After getting the branch set up, go to the Appearance settings (`admin/appearance`).  
2. For the Colorful theme, pick settings and scroll down to 'Color Pairing'. Make sure Firefinch is *not* in that list.
3. Go back to the Appearance Settings and make sure the Traditional theme is set as default.
4. Go to the settings for Traditional and select 'Firefinch' as the 'Color Pairing'. 
5. Go back to the site and make sure it changes color for links, buttons, etc. (It should be a red style color, screenshot below).
6. Go back to the 'Color Pairing' settings, change it to something else like 'Blue Jay', save and make sure it changes color to that. 

## Screenshots
Firefinch colors:
![firefinch](https://user-images.githubusercontent.com/8405274/183469612-19187531-5387-405b-bc7e-ae6167b933ef.png)

Color pairings:
![Screen Shot 2022-08-08 at 12 35 37 PM](https://user-images.githubusercontent.com/8405274/183469655-7b882154-45fb-47d0-b219-811cad682c3e.png)
